src/helpers/userInput.ts

```diff
@@ -13,17 +14,29 @@ export async function newProjectUserInput(): Promise<ProjectModel | undefined> {
   // Project name
   const name = await vscode.window.showInputBox({
     prompt: 'Project Name',
-    value: 'New Project',
+    value: 'NewProject',
     validateInput: (v) => {
-      if (!v.match(/^[^\.].*[^\.]$/)) {
+      if (!v.match(/^[A-Za-z0-9_.-]+$/)) {
```

Comment:
Why you have changed requirements for project name? It is a directory name, converted using `toArtifactId`